types/jsreport-html-to-xlsx/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for jsreport-html-to-xlsx 1.4
+// Type definitions for jsreport-html-to-xlsx 2.0
```

Comment:
When performing a major version upgrade, please follow the steps outlined here: https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version